index.bs

```diff
@@ -744,9 +749,14 @@ never exceed the [=sampling frequency=] for the given [=sensor type=].
 ## Conditions to expose sensor readings ## {#concepts-can-expose-sensor-readings}
 
 The user agent must verify that all [=mandatory conditions=] are satisfied to ensure it
-<dfn>can expose sensor readings</dfn> to a given [=active document=].
+<dfn>can expose sensor readings</dfn> to the {{Sensor}} objects of a certain
+[=sensor type|type=] that belong to a certain [=active document=].
 
 The <dfn>mandatory conditions</dfn> are the following:
+ - The given document is a [=responsible document=] of a [=secure context=].
+ - For each [=permission name=] from the [=sensor type=]'s associated
+   [=sensor permission names=] [=ordered set|set=], the corresponding permission's
+   [=permission state|state=] is "granted".
```

Comment:
[="granted"=]

Reply:
we represent enum values like this through the whole document, so I would keep this consistency so far.